app/javascript/components/upload/H5adStep.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react'
```

Comment:
Blocker: As part of substantive architecture refactor for h5ad -> AnnData:

Let's rename this file to e.g. `AnnDataStep.jsx`.